css/style.css

```diff
@@ -61,9 +134,76 @@ body {
 
 /* Center the container */
 .container {
+  margin: 0 auto;
+  height: 400px;
+  background: url(../images/header_bg.jpg) no-repeat center top rgb(134,141,157);
```

Comment:
:hammer: The background is like 95% there. There's one property you can add to your `.container` class that would make the image "cover" the whole width. Can you think of what it is? Right now the "size" of your background seems limited. Hmmmm....